packages/slate-plugins/src/elements/table/renderElementTable.tsx

```diff
@@ -7,6 +7,7 @@ import { RenderElementTableOptions, TableType } from './types';
 const StyledTable = styled.table`
   margin: 10px 0;
   border-collapse: collapse;
+  width: 100%;
```

Comment:
Full table width fits better as long as we can't position other elements next to the table anyway. Maybe this could be configurable?

Reply:
Yes resizing the table, the columns and the rows would be nice in the future.